bootstrap.sh

```
@@ -1,10 +1,43 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -euo pipefail
+
+# 0) Create/ensure kind cluster from cluster.yml
+if ! kind get clusters | grep -qx "kind"; then
+  kind create cluster --config=cluster.yml
+fi
+
+# 1) Wait for nodes to be Ready (best effort)
+kubectl wait --for=condition=Ready nodes --all --timeout=180s || true
```

The node readiness wait ends with `|| true`, which hides readiness failures and lets the script continue even when nodes are NotReady. Remove `|| true` or replace with a retry loop and clear diagnostics so the script fails early when nodes are not ready. This ensures subsequent labeling/tainting and manifest application are done on a functional cluster (checklist item 3.5).